src/views/Main.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { getBlogs } from '../services/blogs';
+import BlogCard from '../components/BlogCard/BlogCard';
+
+export default function Main() {
+  const [blogs, setBlogs] = useState([]);
+
+  useEffect(() => {
+    const fetchBlogs = async () => {
+      const resp = await getBlogs();
+      setBlogs(resp);
+    };
+    fetchBlogs();
+  }, []);
+
+  return (
+    <main>
+      {blogs.map((blog) => (
+        <BlogCard key={blog.id} {...blog} />
+        // <p key={blog.id}>{blog.title}</p>
+      ))}
```

Comment:
Nice job mapping through each of the blogs and passing them as props with the spread operator. So much easier than writing the key and value pairs as props!